apps/emqx_rule_engine/src/emqx_rule_engine_schema.erl

```diff
@@ -295,6 +295,14 @@ rule_engine_settings() ->
                     deprecated => {since, "v5.0.22"},
                     importance => ?IMPORTANCE_HIDDEN
                 }
+            )},
+        {db_actions_undefined_vars_as_null,
```

Comment:
Q. Won't it be more natural to have this parameter closer to each action's schema, for example somewhere [here](https://github.com/emqx/emqx/blob/46877e979b638ae17ec79464e7af0100ef4e0b72/apps/emqx_connector/src/emqx_connector_schema_lib.erl#L56-L62)? Right now it looks too broad and a bit out of place.

Reply:
changed